backend/templates/auth/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "battling/base.html" %}
```

Comment:
I think you need to refactor the name here and call this `battling/base.html` view something else. It seems like you're not using the `base.html` you already have, right?

Maybe copy the code from `battling/base.html` to `base.html` and use only `base.html`?